### ISSUE TITLE
Share example(s) on Colab

### ISSUE THREAD

**Issue by globien** [mathriddle/ColabTurtlePlus#34]
So impressed by your huge work!

A suggestion: You may consider to share your example(s) on Colab (playground mode), so readers can watch the effects directly.

**Comment by mathriddle** [comment]
Thanks! I'm not familiar with playground mode in Colab, but I'll investigate that. I'm still tweaking the one example currently in my github repo.

**Comment by globien** [comment]
Yeah, it's quite fun and easy.  It's in the File menu of Colab, the second command option: Open in playground. With the file opened in the playground mode you can share it without worrying your original file being changed by readers. They can view, edit and run the program though.

**Comment by mathriddle** [comment]
Ok, I found the playground mode, but I couldn't see how to share the notebook without sending someone the link or adding their names in the share dialog box. I tried sending the link to myself using a second google account I have, but when I clicked the link to the notebook, I was told that access was denied (with a button to request access). So I'm still not sure now to make use of your playground mode suggestion.

But I did figure out how to create an examples folder in my github repo with files that had links to the notebooks that would allow someone to run the notebooks in Colab (assuming they have a Google account) without being able to save changes. It doesn't seem to involve the playground mode, however. 

While I was working on this I also found a few bugs in my python code, so I was fixing those at the same time.
